matrix_client/api.py

```diff
@@ -152,7 +152,8 @@ def login(self, login_type, **kwargs):
             "type": login_type
         }
         for key in kwargs:
-            content[key] = kwargs[key]
+            if kwargs[key]:
```

Comment:
I'm confused by this. We're already iterating through all keys in kwargs. Under what circumstances would `kwargs[key]` be falsey?

All of this is fine, but in case you aren't aware (as I wasn't until a few months ago), `content.update(kwargs)` will do this on a single line with no explicit iteration required.

Reply:
Now reading further on, I see why since we're not validating input in the client method. In light of that, this is fine IMO.